heroku/cli
# refactor(pg-v5): Move command pg:links:destroy to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bCyBPYA0/view

This converts `pg:links:destroy` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:links:destroy --app APP DB1 DB2` with an app that has a postgres db DB1 with a datastore link to DB2
3) Verify the command succeeds in destroying the link
4) Compare with prod
